gadopt/gplates/gplates.py

```diff
@@ -72,6 +77,8 @@ class GplatesVelocityFunction(GPlatesFunctionalityMixin, fd.Function):
         val (optional): Initial values for the function. Defaults to None.
         name (str, optional): Name for the function. Defaults to None.
         dtype (data type, optional): Data type for the function. Defaults to ScalarType.
+        kwargs (dict, optional): Additional keyword arguments, inlcuding `solver_parameters`
+            (dict, optional), and `ds_kwargs` (dict, optional) for the ds measure.
```

Comment:
Aren't you updating `ds_kwargs` directly with the kwargs? I don't think this matches the code...